src/modules/displayData.js

```diff
@@ -0,0 +1,86 @@
+import removeList from './removeList.js';
+import editList from './editList.js';
+import completedList from './completedList.js';
+
+const input = document.querySelector('.todo-input');
+const addDesc = document.querySelector('#add-task');
+let getLocalStorage = localStorage.getItem('AppData') !== null ? JSON.parse(localStorage.getItem('AppData')) : [];
+const li = document.querySelector('#todoList');
+let index = getLocalStorage.length;
+const clear = document.querySelector('.clear');
+
+export default function DisplayUI() {
+  if (getLocalStorage !== null) {
+    li.innerHTML = '';
+    getLocalStorage.forEach((todo) => {
+      li.innerHTML += `
+        <li class="listItems">
+          <input type="checkbox" class="todo-check" id="checked"  value="${todo.completed}" ${todo.completed === true ? 'checked' : ''}>
+          <input type="text" class="description ${todo.completed === true ? 'checked' : ''}" value="${todo.desc}">
+          <i class="fa-solid fa-ellipsis-vertical dots"></i>
+          <button type="button" id="delete"><i class="fa fa-trash"></i></button>
+        </li>
+      `;
+    });
+  }
+
+  for (let i = 0; i < li.querySelectorAll('.listItems').length; i += 1) {
+    const listItems = li.querySelectorAll('.listItems')[i];
+    // eslint-disable-next-line no-loop-func
```

Comment:
With this inline eslint comment, you're indirectly changing the linter config. It would be nice to solve the error instead of bypassing it with an inline comment.

Also apply to line - 41 and 61.